server/src/pages/shotindex/model.js

```diff
@@ -28,7 +28,7 @@ exports.createModel = function(req) {
       image.url = createProxyUrl(req, image.url);
     }
   }
-  if (shots !== null && shots !== undefined) {
+  if (shots && shots.length) {
```

Comment:
Would we want to use [`Array.isArray()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray)? Although, I can't imagine that `shots` would ever be a string and we'd get odd behavior w/ it checking a string length.

Reply:
ah, got merged before I had a chance to fix it up ^_^